mosquitto.c

```diff
@@ -1147,7 +1147,12 @@ PHP_MOSQUITTO_API void php_mosquitto_message_callback(struct mosquitto *mosq, vo
 
 	object_init_ex(message_zval, mosquitto_ce_message);
 	message_object = mosquitto_message_object_from_zend_object(Z_OBJ_P(message_zval));
-	mosquitto_message_copy(&message_object->message, message);
+	message_object->message.mid = message->mid;
```

Comment:
This caused a leak? Was it logged by PHP's memory leak detection or was it something you found in Valgrind or similar?